src/System.Linq.Expressions/src/System/Linq/Expressions/Interpreter/Utilities.cs

```diff
@@ -144,8 +144,8 @@ internal static object GetPrimitiveDefaultValue(Type type)
                     return default(float);
                 case TypeCode.Double:
                     return default(double);
-                //case TypeCode.DBNull:
-                //    return default(DBNull);
+                case TypeCode.DBNull:
+                    return default(DBNull);
```

Comment:
See https://github.com/dotnet/corefx/pull/13952 for the `GetTypeCode` replacement.